AndroidSDK/src/com/leanplum/NewsfeedMessage.java

```diff
@@ -149,6 +149,9 @@ public Date deliveryTimestamp() {
    * Returns the delivery timestamp of the newsfeed message.
    */
   public Date getDeliveryTimestamp() {
+    if (deliveryTimestamp == null) {
+      return new Date();
```

Comment:
- yes, but then guard against null here: https://github.com/Leanplum/Leanplum-Android-SDK/pull/27/files#diff-f7185a5c0a8b80ab803fcb5bd4bf2c50R321
and in all other invocations of this method.
- also we should update docs, to document that it might return null.